python/cudf/cudf/utils/ioutils.py

```diff
@@ -83,24 +83,30 @@
 doc_read_avro = docfmt_partial(docstring=_docstring_read_avro)
 
 _docstring_read_parquet_metadata = """
-Read a Parquet file's metadata and schema
+Read a Parquet file's metadata
 
 Parameters
 ----------
 path : string or path object
     Path of file to be read
+fs : fsspec.AbstractFileSystem, optional
+    fsspec file-system object.
+storage_options : dict, optional
+    Extra options that make sense for a particular storage connection,
+    e.g. host, port, username, password, etc. For HTTP(S) URLs the
+    key-value pairs are forwarded to urllib as header options. For
+    other URLs (e.g. starting with “s3://”, and “gcs://”) the key-value
+    pairs are forwarded to fsspec. Please see fsspec and urllib for more details.
 
 Returns
 -------
-Total number of rows
-Number of row groups
-List of column names
+pyarrow.parquet.FileMetaData
```

Comment:
Do we have precedent for returning pyarrow types? Probably not an issue, just checking.

Reply:
This is a good question - There is no precedent that I know about, but we would need a new C++/Cython API to return a cudf-based object with the same kind of information.  There may be some performance benefits to this (perhaps more-efficient metadata aggregation across multiple files), but that overall effort may not gain us much beyond adding more code to maintain.  Another alternative may be to return  a `pd/cudf.DataFrame` summary of the Parqet metadata and encourage Pandas to adopt the same API.

On a related note, I am struggling a bit with the question of whether this API should exist in cudf at all, because (1) we are just returning a `FileMetaData` object without using cudf in any meaningful way, and (2) this API doesn't even exist in Pandas.